### ISSUE TITLE
Secretgen-controller deployment failed: namespace not found

### ISSUE THREAD

**Issue by shwethaks1** [carvel-dev/secretgen-controller#63]
**What steps did you take:**
```
C:\Users\tkgadmin>kubectl apply -f https://github.com/vmware-tanzu/carvel-secretgen-controller/releases/download/v0.8.0/release.yml
customresourcedefinition.apiextensions.k8s.io/secretexports.secretgen.carvel.dev created
customresourcedefinition.apiextensions.k8s.io/secretimports.secretgen.carvel.dev created
customresourcedefinition.apiextensions.k8s.io/certificates.secretgen.k14s.io created
customresourcedefinition.apiextensions.k8s.io/passwords.secretgen.k14s.io created
customresourcedefinition.apiextensions.k8s.io/rsakeys.secretgen.k14s.io created
customresourcedefinition.apiextensions.k8s.io/sshkeys.secretgen.k14s.io created
namespace/secretgen-controller created
serviceaccount/secretgen-controller-sa created
clusterrole.rbac.authorization.k8s.io/secretgen-controller-cluster-role created
clusterrolebinding.rbac.authorization.k8s.io/secretgen-controller-cluster-role-binding created
Error from server (NotFound): error when creating "https://github.com/vmware-tanzu/carvel-secretgen-controller/releases/download/v0.8.0/release.yml": namespaces "secretgen-controller" not found
```

**Anything else you would like to add:**
Deployment worked in secretgen-controller `v0.7.1`

**Environment:**

- Windows
- secretgen-controller version: `v0.8.0`
- Kubernetes version: `v1.22.1`

---
Vote on this request

This is an invitation to the community to vote on issues, to help us prioritize our backlog. Use the "smiley face" up to the right of this comment to vote.

👍 "I would like to see this addressed as soon as possible"
👎 "There are other more important things to focus on right now"

We are also happy to receive and review Pull Requests if you want to help working on this issue.

**Comment by joe-kimmel-vmw** [comment]
Thanks @shwethaks1 for this bug - we will try to post a fix. in the meanwhile you should be able to deploy using `kapp`

**Comment by benmoss** [comment]
I think the fix will be to prefix the namespace file with `0-` so it gets ordered earlier in the manifest: https://github.com/vmware-tanzu/carvel-kapp-controller/tree/develop/config

**Comment by cppforlife** [comment]
i believe it should be resolved in v0.8.1+
